package.json

```diff
@@ -31,6 +31,8 @@
   },
   "homepage": "https://github.com/neomad-team/neomad.org#readme",
   "dependencies": {
+    "preact": "^8.1.0",
+    "preact-compat": "^3.16.0",
     "react": "^15.5.4",
```

Comment:
If *you* like to use them but they're not *required* for the project, you should not insert them as dependencies.
You may still install them for yourself (`npm install react` without `--save`).
If it's mandatory for all developers but not for running the project, then [use devDependencies](https://docs.npmjs.com/files/package.json#devdependencies)